src/sql/base/browser/ui/table/media/slickColorTheme.css

```diff
@@ -74,17 +74,12 @@
 	border-color: var(--color-cell-border-active);
 }
 
-.vs slick-grid.active .grid .slick-cell.selected {
-	background-color: var(--color-cell-bg-grid-selected);
-}
-
-/* EditData versions */
-
-.vs .grid .slick-cell .active {
+/* EditData version */
+.vs .slickgridContainer .grid .slick-cell .active {
```

Comment:
If this is truly meant for the edit data grid only then it should really go in editData.css I'd think.

But given that there doesn't seem to be a reason for these to apply to edit data only (aside from the fact that edit data is currently the only grid with these classes) I think it'd be preferable to just combine them here so we don't have so much duplication for the rules that are meant to be for the same thing just in the different grids. e.g.

```
.vs .slickgridContainer .grid .slick-cell .active,
.vs slick-grid.active .grid .slick-cell .active { 
     border-color: var(--color-cell-border-active);
}
```